src/stories/text/Text.tsx

```diff
@@ -1,22 +1,37 @@
 import type { ComponentProps, FC, ReactNode } from 'react';
 import { createElement } from 'react';
 import clsx from 'clsx';
-import styles from './Text.module.scss';
 import typography from './Typography.module.css';
+import styles from './Text.module.scss';
```

Comment:
@heysanil Added this to apply the Text stylings last, so that they take priority. Let me know if this should be ok - or if we rather do another method (maybe we can apply these text stylings elsewhere or a separate file simply).